docker-compose-confluent-cloud.yml

```diff
@@ -22,4 +22,10 @@ services:
       KAFKA_TYPE: ${KAFKA_TYPE}
       CONFLUENT_KEY: ${CONFLUENT_KEY}
       CONFLUENT_SECRET: ${CONFLUENT_SECRET}
+      KAFKA_ENABLE_AUTO_COMMIT: ${KAFKA_ENABLE_AUTO_COMMIT}
+      KAFKA_AUTO_COMMIT_INTERVAL_MS: ${KAFKA_AUTO_COMMIT_INTERVAL_MS}
+      KAFKA_SESSION_TIMEOUT_MS: ${KAFKA_SESSION_TIMEOUT_MS}
+      GOOGLE_APPLICATION_CREDENTIALS: '/google/gcp_service_account.json'
```

Comment:
Is this google credential value: ${GOOGLE_APPLICATION_CREDENTIALS}

Reply:
Yes. The value of the environment variable is mounted to `/google/gcp_service_account.json` and then that file is referenced in the application as GOOGLE_APPLICATION_CREDENTIALS.